README.md

```diff
@@ -163,6 +163,55 @@ helm tiller run \
 -    --set container.image=eu.gcr.io/census-eq-ci/eq-survey-runner-benchmark:latest
 ```
 
+## Visualising Benchmark Results
+
+You can use the `visualise_results.py` script to visualise benchmark results over time. 
+
+### Pre-Requisites
+
+You need to be authenticated with GCP in order to download the benchmark results. To do this use:
+
+`gcloud auth login --project <project_id>`
+
+`gcloud auth application-default login`
+
+Alternatively you can use the project service Account:
+
+- Go to the `eq-daily-performance-test` project in GCP
+- Go to IAM&Admin and open up the Service Accounts tab
+- Click on the `results-analysis` service account
+- Click edit, then, click 'Create Key'
+- Select JSON and store the file in a safe location - the path for this file will be used as an environment variable for the `get_benchmark_results.py` script.
```

Comment:
This is specific to the project. I think all we need to say here is that you can use a service account which must have the necessary permission on the bucket.